packages/camera/camera_android/example/pubspec.yaml

```diff
@@ -8,13 +8,9 @@ environment:
 
 dependencies:
   camera_android:
-    # When depending on this package from a real application you should use:
-    #   camera_android: ^x.y.z
-    # See https://dart.dev/tools/pub/dependencies#version-constraints
-    # The example app is bundled with the plugin so we use a path dependency on
-    # the parent directory to use the current plugin's version.
```

Comment:
Why is this comment removed? We put this in examples deliberately.

Reply:
Restored comments for all packages

Reply:
Please remove the comma you added (here and in the other packages) so that it's not slightly different in these packages as compared to all our other packages. Having boilerplate be identical is a feature.

Reply:
comma removed